packages/nouns-webapp/src/components/CandidateSponsors/SignatureForm.tsx

```diff
@@ -75,7 +76,8 @@ const SignatureForm = (props: Readonly<SignatureFormProps>) => {
   const [isTxSuccessful, setIsTxSuccessful] = useState(false);
   const [errorMessage, setErrorMessage] = useState<ReactNode>('');
 
-  const chainId = useChainId();
+  const chainId = defaultChain.id;
```

Comment:
could use `CHAIN_ID` for consistency, and also it's more explicitly a static (build time) config. `defaultChain.id` has adds more indirections

Reply:
`defaultChain` isn’t just a chain id; it’s packed with metadata we can plug in later to DRY up our code, from explorer links to any other chain-specific details.

Reply:
nitpick, but should probably pick one or another for consistency.
(CHAIN_ID is used right below this one)

Reply:
Fix this specific one, but if I update all of it's usage its going to trigger a-lot of lint errors, trying to keep PRs small as possible moving forward. 😹